roles/openshift_examples/examples-sync.sh

```diff
@@ -26,6 +31,6 @@ cp nodejs-ex-master/openshift/templates/* ${EXAMPLES_BASE}/quickstart-templates/
 cp dancer-ex-master/openshift/templates/* ${EXAMPLES_BASE}/quickstart-templates/
 cp cakephp-ex-master/openshift/templates/* ${EXAMPLES_BASE}/quickstart-templates/
 mv application-templates-master/jboss-image-streams.json ${EXAMPLES_BASE}/xpaas-streams/
-find application-templates-master/ -name '*.json' -exec mv {} ${EXAMPLES_BASE}/xpaas-templates/ \;
+find application-templates-master/ -name '*.json' ! -wholename '*secret*' -exec mv {} ${EXAMPLES_BASE}/xpaas-templates/ \;
```

Comment:
The secrets need to be per project so shouldn't be imported into the openshift namespace. We also came to the conclusion that end users wouldn't have access to /usr/share/openshift/examples on the master so there wasn't a lot of value in distributing them there for manual use.

Unfortunately it means the xpaas examples that rely on secrets aren't usable without users knowing to create them and we don't have a great way to instruct them to do so and/or deliver these files to users who don't have access to the master.

There's an ongoing discussion about how to improve that.